connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/Encoders.scala

```diff
@@ -176,6 +177,32 @@ object Encoders {
    */
   def row(schema: StructType): Encoder[Row] = RowEncoderFactory.encoderFor(schema)
 
+  /**
+   * (Scala-specific) Creates an encoder that serializes objects of type T using generic Java
+   * serialization. This encoder maps T into a single byte array (binary) field.
+   *
+   * T must be publicly accessible.
+   *
+   * @note
+   *   This is extremely inefficient and should only be used as the last resort.
+   * @since 1.6.0
```

Comment:
Should it be 4.0.0?

Reply:
It does not really matter, since this file will be merged with the one in sql/core in a couple of weeks.